### PR TITLE
Add an option to add/update icons to entries without changing last modified

### PR DESCRIPTION
**Pull request by HypervisorX**
Applying this adds a menu item and supporting configuration for the option to enable/disable
update of last modification time during entry modification. The default
configuration is to update the last modification time.

I tested it out on windows and have been using it without issue for a few days. Feel free to look it over and accept it, reject it, or change it, but I figured there are probably other people that might have a similar problem and figured I would try to contribute back in case you wanted it.

**Why?:** I find your icon downloader addon much nicer than a different one I used but I missed being able to get an icon without changing the entry last modified date. I use the last modified of entries to keep track of the last time i changed a password or the entry, so I needed a way to get all the icons for the entries in my database without ending up with an entire database of entries that had today's date as last modified just because they had an icon added to them. This is the solution I came up with. I'm sure some people find the last time the icon on their password entry was added or updated relevant, but to me it doesn't matter and makes the last modified column unhelpful for my personal purposes. 

**Edit:** Screenshot with the extra menu option shown below. If you do end up including this I would probably rename that option, it's a bit wordy and long right now.

favicon

### CODE REVIEW

**Comment by navossoc** [comment]
Great! I'll take a closer look tomorrow.

**Comment by navossoc** [comment]
> Edit: Screenshot with the extra menu option shown below. If you do end up including this I would probably rename that option, it's a bit wordy and long right now.

I was looking at this just now.
I think `Update entry last modification time` should be enough, isn't?

Probably I will add an page on the wiki to explain in more details how this option works.
Since it will update the last modification time only if this option is enabled and if there is a change on the favicon, otherwise nothing will be touched.

**Comment by HypervisorX** [comment]
> I think `Update entry last modification time` should be enough, isn't?

Yes, I agree that is much better.

**Comment by navossoc** [comment]
@HypervisorX it is available here:
https://github.com/navossoc/KeePass-Yet-Another-Favicon-Downloader/releases/tag/v1.2.4.0-pre

I will make the official release on monday if everything goes right.
